runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodePlugin.java

```diff
@@ -192,12 +198,20 @@ public MotherlodeSession getSession()
 		return session;
 	}
 
+	private int lastSackValue;
```

Comment:
Can't be a local variable because I am tracking a change in a specific varbit. I was told in Discord that you need to store the previous value. If there is another way to check for a change in this particular varbit I'm all ears.